src/features/search/components/SearchResults.tsx

```diff
@@ -61,6 +62,8 @@ export const SearchResults: React.FC = () => {
   const { navigate } = useNavigation<UseNavigationType>()
   const { section } = useLocationType(searchState)
   const { label: locationLabel } = useLocationChoice(section)
+  const { data: appSettings } = useAppSettings()
+  const filtersButtonsDisplay = appSettings?.appEnableCategoryFilterPage ?? false
```

Comment:
`shouldDisplayFiltersButtons` plutôt ? Ou n'importe quel nom qui garde l'idée que c'est un booléen :)

Reply:
Oui, `should|has|is` sont des suffixes courant pour les booleans, prend juste une notation standard, https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig